recipe/build-libadios.sh

```diff
@@ -89,14 +66,14 @@ cmake              \
     -DADIOS2_HAVE_ZFP_CUDA=OFF                \
     -DADIOS2_BUILD_EXAMPLES=OFF               \
     -DADIOS2_RUN_INSTALL_TEST=OFF             \
-    -DPython_EXECUTABLE:FILEPATH=${PYTHON}    \
-    -DPython_INCLUDE_DIR=$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])") \
-    -DCMAKE_INSTALL_LIBDIR=lib        \
-    -DCMAKE_INSTALL_PREFIX=${PREFIX}  \
-    -DKWSYS_LFS_WORKS=0               \
-    -DPNG_PNG_INCLUDE_DIR=${PREFIX}
+    -DPython_EXECUTABLE:FILEPATH="${PYTHON}"  \
+    -DPython_INCLUDE_DIR="$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])")" \
+    -DCMAKE_INSTALL_LIBDIR=lib                \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}"        \
+    -DPNG_PNG_INCLUDE_DIR="${PREFIX}"
+
+cmake --build build -j${CPU_COUNT} -v
```

Comment:
```suggestion
cmake --build build "-j${CPU_COUNT}" -v
```
I know that this is unneeded and a similar argument that can be done with the ${} in the variables. The reason for introducing this is that this PR is also trying to make this file shellcheck (as wrote in the description) compatible, and shellcheck requires using double quotation by default unless there is an explicit intention not to do it (splitting the variable during expansion). I really wish it to pass shellcheck if you feel strong about it we can have a disable statement for this above.

Reply:
@ax3l

Reply:
That's ok - fine by me :)